### PR TITLE
Google Cloud Storage - Update gcs-list-bucket-objects to support prefix and delimiter arguments.

### PR DESCRIPTION
**Pull request by zdrouse**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
The google cloud storage python client supports these arguments.
https://github.com/googleapis/python-storage/blob/main/samples/snippets/storage_list_files_with_prefix.py

## Description
Add support prefix and delimiter support for gcs-list-bucket-objects command.  This will allow users to limit blobs returned from a GCS bucket.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/GoogleCloudStorage/Integrations/GoogleCloudStorage/GoogleCloudStorage.yml [hunk 1]
```diff
@@ -116,6 +116,11 @@ script:
     description: Deletes a bucket.
     name: gcs-delete-bucket
   - arguments:
+      defaultValue: None
+      description: use a delimiter if you want to limit results within a specific "folder", default is None
+      name: delimiter
+      description: specify to limit blobs within a "folder" i.e. "folder-1/" if blob is "folder-1/file.txt"
+      name: prefix
```
Comment (adi88d) [inline Packs/GoogleCloudStorage/Integrations/GoogleCloudStorage/GoogleCloudStorage.yml:123]:
```suggestion
       description: use a delimiter if you want to limit results within a specific "folder", default is None
    - name: delimiter
       description: specify to limit blobs within a "folder" i.e. "folder-1/" if blob is "folder-1/file.txt"
    - name: prefix
```
please remove the default value and fix the yml structure

**Review by adi88d — CHANGES_REQUESTED** [review]
Great Job!
Thanks a lot for this contribution, It is highly appreciated.
I added some review comments and suggestion:
- remove the `Packs/GoogleCloudStorage/CONTRIBUTORS.json` file
- update release notes for the pack using [demisto-sdk update-release-notes](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/update_release_notes/README.md) command
- add to the delimiter and prefix arguments to`GoogleCloudStorage/README.md`

**Review by zdrouse — COMMENTED** [review]
I committed the suggestion for YAML file although I'm not sure I understand it since all the other arguments and commands already implemented don't follow it

File: Packs/GoogleCloudStorage/Integrations/GoogleCloudStorage/GoogleCloudStorage.yml [hunk 2]
```diff
@@ -119,6 +119,10 @@ script:
     - default: true
       description: Name of the bucket in which to list objects. If not specified, operation will be performed on the default bucket parameter.
       name: bucket_name
+    - description: Specify to limit blobs within a "folder" i.e. "folder-1/" if blob is "folder-1/file.txt".
```
Comment (ShirleyDenkberg) [inline Packs/GoogleCloudStorage/Integrations/GoogleCloudStorage/GoogleCloudStorage.yml:122]:
```suggestion
    - description: Specify to limit blobs within a "folder" i.e., "folder-1/" if blob is "folder-1/file.txt".
```

File: Packs/GoogleCloudStorage/Integrations/GoogleCloudStorage/GoogleCloudStorage.yml [hunk 3]
```diff
@@ -119,6 +119,10 @@ script:
...
+      name: prefix
+    - description: Use a delimiter if you want to limit results within a specific "folder" and without any nested blobs i.e. "/".
```
Comment (ShirleyDenkberg) [inline Packs/GoogleCloudStorage/Integrations/GoogleCloudStorage/GoogleCloudStorage.yml:124]:
```suggestion
    - description: Use a delimiter if you want to limit results within a specific "folder" and without any nested blobs i.e., "/".
```

File: Packs/GoogleCloudStorage/Integrations/GoogleCloudStorage/README.md [hunk 4]
```diff
@@ -291,6 +291,16 @@
 <td style="width: 421px;">Name of the bucket in which to list objects.</td>
 <td style="width: 103px;">Required</td>
 </tr>
+<tr>
+<td style="width: 184px;">prefix</td>
+<td style="width: 421px;">Specify to limit blobs within a "folder" i.e. "folder-1/" if blob is "folder-1/file.txt".</td>
```
Comment (ShirleyDenkberg) [inline Packs/GoogleCloudStorage/Integrations/GoogleCloudStorage/README.md:296]:
```suggestion
<td style="width: 421px;">Specify to limit blobs within a "folder" i.e., "folder-1/" if blob is "folder-1/file.txt".</td>
```

File: Packs/GoogleCloudStorage/Integrations/GoogleCloudStorage/README.md [hunk 5]
```diff
@@ -291,6 +291,16 @@
...
+<td style="width: 103px;">Optional</td>
+</tr>
+<tr>
+<td style="width: 184px;">delimiter</td>
+<td style="width: 421px;">Use a delimiter if you want to limit results within a specific "folder" and without any nested blobs i.e. "/".</td>
```
Comment (ShirleyDenkberg) [inline Packs/GoogleCloudStorage/Integrations/GoogleCloudStorage/README.md:301]:
```suggestion
<td style="width: 421px;">Use a delimiter if you want to limit results within a specific "folder" and without any nested blobs i.e., "/".</td>
```

File: Packs/GoogleCloudStorage/ReleaseNotes/1_0_12.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+
+##### Google Cloud Storage
+
+- Added the following arguments to the **gcs-list-bucket-objects** command:
+  - ***delimiter***
+  - ***prefix***
```
Comment (ShirleyDenkberg) [inline Packs/GoogleCloudStorage/ReleaseNotes/1_0_12.md:8]:
```suggestion
  - *delimiter*
  - *prefix*
```

File: Packs/GoogleCloudStorage/ReleaseNotes/1_0_12.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+
+##### Google Cloud Storage
+
+- Added the following arguments to the **gcs-list-bucket-objects** command:
```
Comment (ShirleyDenkberg) [inline Packs/GoogleCloudStorage/ReleaseNotes/1_0_12.md:6]:
```suggestion
- Added the following arguments to the ***gcs-list-bucket-objects*** command:
```

**Pull request merged**